solutions/usaco-915.mdx

```diff
@@ -9,6 +9,44 @@ author: Sathvik Chundru
 
 <LanguageSection>
  
+<PySection>
+
+```py
+import sys
+sys.stdin = open('herding.in', 'r')
+sys.stdout = open('herding.out', 'w')
+cows = list(map(int, input().split()))
+cows.sort()
+a = cows[0]
+b = cows[1]
+c = cows[2]
```

Comment:
there's only 3 cows for this problem right
if so, you can replace lines 18 through 22 with just
```py
a, b, c = map(int, input().split())
```

Reply:
oops. i tried doing that earlier, but i accidently put a list in front of it and it gave me an error. thanks! :)